spec/21-http_response_header_format.md

```diff
@@ -103,10 +103,11 @@ The second least significant bit of the trace-flags field denotes the random-tra
 If a trace was started by a downstream participant and it responds with the `traceresponse` HTTP header, an upstream participant can use this flag to determine if the `trace-id` was generated as per
 the specification for this flag.
 
-The format and requirements for this are the same as those of the random-trace-id flag in the trace-flags field in the `traceparent` request header.
+When a participant starts or restarts a trace (that is, when the participant generates a new `trace-id`), the requirements for this flag are the same as those for the random-trace-id flag in the trace-flags field in the `traceparent` request header. For details, see the section [Random Trace ID Flag](#random-trace-id-flag).
```

Comment:
1. I dropped the phrase "the format" -- since section is about an individual flag, there is no need to explain the format. We state that the traceresponse flags field is a hex encoded 8-bit field at the start of the trace-flags section. Flags don't have an individual format.
2. I added a new constraint to this sentence: The rules from traceparent/random-trace-id flags are now only applied when the participant starts the trace. In my opinion, we cannot just refer to traceparent/random-trace-id for the situation when continuing a trace, because the a trace can be continued from upstream or from downstream, and different rules need to be applied. See below.